FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/DataConverter/DefaultDicomSchemaConverter.cs

```diff
@@ -100,38 +92,61 @@ private JObject ProcessDicomMetadataObject(JToken metadata, ParquetSchemaNode sc
                 var subNodeKeyword = subNodePair.Key;
                 var subNode = subNodePair.Value;
 
-                // Ignore DICOM metadata node if it doesn't exist in schema
-                // Ignore SQ
-                // Ignore empty tag, BulkDataURI and InlineBinary
-                if (subNodeKeyword == null ||
-                    !jObject.ContainsKey(DicomConstants.Value) ||
-                    jObject[DicomConstants.Value] is not JArray)
-                {
-                    continue;
-                }
-
-                var subValueArray = jObject[DicomConstants.Value] as JArray;
-
-                if (subNode.IsRepeated)
+                if (item.Value is JValue jValue)
```

Comment:
It might be better to add some check here, to see whether the item is our self-defined tag (the check condition could be whether the tag starts with "_"). If not, we can throw exceptions to tell customers it's not a valid DICOM tag, since each DICOM tag should be a JObject with child "vr" ([ref](https://dicom.nema.org/medical/dicom/current/output/chtml/part18/sect_F.2.2.html)).

Reply:
Updated the check condition